static/app/components/charts/eventsChart.tsx

```diff
@@ -386,9 +386,13 @@ class EventsChart extends React.Component<EventsChartProps> {
     // Include previous only on relative dates (defaults to relative if no start and end)
     const includePrevious = !disablePrevious && !start && !end;
 
+    let yAxisLabel = isEquation(yAxis) ? getEquation(yAxis) : yAxis;
+    if (yAxisLabel.length > 60) {
+      yAxisLabel = yAxisLabel.substr(0, 60) + '...';
```

Comment:
60 for truncation was chosen pretty arbitrarily, just based on what looked good from testing